### ISSUE TITLE
Hyperparameter Tuning of Deep Neural Network for Regression Task

### ISSUE THREAD

**Issue by pykao** [facebook/Ax#481]
Hi,

I tried to tune the hyperparameter of deep neural network for regression task. I refer to your tutorial, and it set the `object_name='accuracy' for the classification task.

```
best_parameters, values, experiment, model = optimize(
    parameters=[
        {"name": "lr", "type": "range", "bounds": [1e-6, 0.4], "log_scale": True},
        {"name": "momentum", "type": "range", "bounds": [0.0, 1.0]},
    ],
    evaluation_function=train_evaluate,
    objective_name='accuracy',
)
```

However, the task I have is regression task, and the object is to find the smallest mean square error (MSE). Do you know how to approach this problem?

Best,
Ken

**Comment by ldworkin** [comment]
Hi @pykao , have you taken a look at this tutorial? https://ax.dev/tutorials/gpei_hartmann_loop.html 

You can use any function you want for the evaluation, and it can return any metric!

**Comment by pykao** [comment]
Thank you for your reply @ldworkin. I am going to take a look at the tutorial.
